unified-codes/apps/web/stories/EntityBrowser.stories.tsx

```diff
@@ -72,13 +75,16 @@ export const withApolloData = () => {
         .catch((error) => {
           showAlert(error.message, 'error');
         });
+    } else {
+      showAlert('Unable to load, no data service URL specified', 'error');
     }
-    return (
-      <Snackbar open={alert.show} autoHideDuration={6000} onClose={hideAlert}>
-        <Alert onClose={hideAlert} severity={alert.severity}>
-          {alert.text}
-        </Alert>
-      </Snackbar>
-    );
   }
+
+  return (
+    <Snackbar open={alert.show} autoHideDuration={6000} onClose={hideAlert}>
+      <Alert onClose={hideAlert} severity={alert.severity}>
+        {alert.text}
+      </Alert>
+    </Snackbar>
+  );
```

Comment:
Am refactoring this component to include all the alert logic etc... but can resolve this at merge, so come in!

![comein](https://user-images.githubusercontent.com/43223637/90367912-64a1fd80-e0bd-11ea-9568-1a5507c95024.gif)